tests/tests/tier0/bluechi-daemon-reload/main.fmf

```diff
@@ -0,0 +1,2 @@
+summary: Test daemon reload 
```

Comment:
Please change to "Test bluechictl daemon-reload", it needs to be clear from the test summary, what this test covers